src/catalog/src/durable/upgrade/tests.rs

```diff
@@ -19,22 +18,28 @@ use mz_storage_types::sources::SourceData;
 use crate::durable::objects::state_update::StateUpdateKindJson;
 use crate::durable::upgrade::AllVersionsStateUpdateKind;
 
-static PROTO_DIRECTORY: LazyLock<String> =
-    LazyLock::new(|| format!("{}/protos", env!("CARGO_MANIFEST_DIR")));
+const PROTO_DIRECTORY: &str = {
+    if mz_build_tools::is_bazel_build() {
+        "src/catalog/protos"
+    } else {
+        "protos"
+    }
+};
```

Comment:
Not great, but possible to live with I think!

Reply:
Yeah FWIW this is my least favorite part, I'll follow up with this!